frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/SimpleGraphTable.java

```diff
@@ -47,7 +53,7 @@ private SimpleGraphTable(GraphTableModel graphTableModel) {
     setRowSelectionAllowed(true);
     setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
 
-    setDefaultRenderer(BranchOrCommitCell.class, BranchOrCommitCellRendererComponent::new);
+    setDefaultRenderer(BranchOrCommitCell.class, tableCellRenderer);
```

Comment:
Does it need to be a field? Maybe just local var?

Reply:
Ok, it can be, even without local var